src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -17,37 +19,52 @@ public void testCountIn() {
 
 	@Test
 	public void testCallCheck() {
-		defaultCountingOutRhymer rhymer = new defaultCountingOutRhymer();
+		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
 		boolean result = rhymer.callCheck();
-		Assert.assertEquals(true, result);
+		Assert.assertTrue(result);
 
 		rhymer.countIn(888);
 
 		result = rhymer.callCheck();
-		Assert.assertEquals(false, result);
+		Assert.assertFalse(result);
 	}
 
 	@Test
 	public void testIsFull() {
-		defaultCountingOutRhymer rhymer = new defaultCountingOutRhymer();
-		final int STACK_CAPACITY = 12;
-		for (int i = 0; i < STACK_CAPACITY; i++) {
+		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+		int stackCapacity = 0;
+		try {
+			Field maxSize = rhymer.getClass().getDeclaredField("MAX_SIZE");
+			maxSize.setAccessible(true);
+			stackCapacity = maxSize.getInt(rhymer);
+		} catch (Exception e) {
+			fail("testIsFull() - can't access MAX_SIZE field using reflection");
+		}
```

Comment:
Good idea, but using reflection does not indicate a good design. It would be probably better to modify the visibility of some constants